src/lua/luaLoggerBinding.c

```diff
@@ -904,22 +904,23 @@ static int lua_get_virtual_channel(lua_State *ls)
          * that case first
          */
         if (lua_isnumber(ls, 1)) {
-        		VirtualChannel *vc;
+                VirtualChannel *vc;
                 vc = get_virtual_channel(lua_tointeger(ls, 1));
                 if (vc) {
-						lua_pushnumber(ls, vc->currentValue);
-						return 1;
+						                  lua_pushnumber(ls, vc->currentValue);
+						                  return 1;
                 }
         } else {
-        		struct sample * s = get_current_sample();
-				double value;
-				if (s && get_sample_value_by_name(s, lua_tostring(ls, 1), &value)){
-						lua_pushnumber(ls, value);
-						return 1;
-        		}
-		}
-
-		return luaL_error(ls, "Channel not found!");
+                struct sample * s = get_current_sample();
+				            double value;
+				            char * units;
+				            if (s && get_sample_value_by_name(s, lua_tostring(ls, 1), &value, &units)){
+						                  lua_pushnumber(ls, value);
+						                  return 1;
```

Comment:
What  is up with the indentation here?

Reply:
IDK, it's correct in my editor with spaces :-/